### DISCUSSION TITLE
Disable graying out hidden or minimized windows

### DISCUSSION THREAD

**Discussion by Romasato** [lawand-dot-io/taskbar#44] [Feature requests]
Hi there,
As promised few days ago on Reddit, here I am with a feature request :)

Would it be possible to add a checkbox to disable graying out tasks representing hidden windows? 

Knowing whether window is hidden/minimized or not does not really provide any useful info  - especially given most of my windows are kept in maximized state. The grayed out window task implies the window is disabled/unresponsive more than anything else.

What do you think?

Thanks again for creating this app!

**Comment by lawand-dot-io** [comment]
Hi. Interesting idea, shouldn't be too hard to implement. Will consider it for one of the next versions. Cheers :)

**Comment by lawand-dot-io** [comment]
This feature has been implemented in Taskbar version 1.2.1 which is now released. Closing the ticket.

@Romasato

**Reply by Romasato** [reply]
This is awesome @lawand-dot-io! 🤩 Thank you so much!

**Reply by lawand-dot-io** [reply]
you're welcome, please report any bugs if you find any. cheers

**Comment by ChrissyTopal** [comment]
I did not want to open a new ticket, but do you think it is possible to turn off the animation altogether? When clicking on a window multiple times, nothing should happen. I sometimes click double on a window, then the animation plays, then i have to open it again (animation again).

Thank you very much.

**Reply by lawand-dot-io** [reply]
Hello. Which animation do you mean? The minimize/unminimize animation?

**Reply by ChrissyTopal** [reply]
Hello, thanks for your reply!

Yes that's what I mean! The animation where it looks like being sucked in when you close the window, after clicking on the open window

**Reply by lawand-dot-io** [reply]
I see. Have a look at the discussion here and let me know what you think: https://github.com/lawand-dot-io/taskbar/discussions/60
